test/erc721/proof.js

```diff
@@ -88,18 +88,18 @@ module.exports = {
             tokenRole.salt,
         ],
         proofs:[
-            grossAmount.sorted_hashes,
-            currency.sorted_hashes,
-            due_date.sorted_hashes,
-            sender.sorted_hashes,
-            status.sorted_hashes,
+            grossAmount.sorted_hashes.slice(0,-1),
```

Comment:
I think it would be nice to actually modify the proofs in the file instead of doing this at test runtime. Or at least add a note why this is being done right now and that it could be cleaned up in the future.

Reply:
I will add a comment. I think it should be easy to replace the json.